src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] as the previous reviewer state you should change your `App`component from function to class-based so please try to do that in order to meet the requirements

Reply:
@youmari  I'm sorry but what should I do I have already done that.. please explain what exactly I should do I have already converted the app function component to a class component.

Reply:
@youmari PLEASE view the changes made in the previous commit I made the page responsive and changed the app component to a class, as I only have one review left.

Reply:
@youmari and if there are any changes required also let me know in the comments.

Reply:
@OkoyeCharles You should change the app function to class-based  
For example : 
```JS
class App extends React.Component {
}
```

Reply:
@youmari but that is what i did

Reply:
@youmari can we call on zoom so i could show you what i have done

Reply:
@youmari  below is my current App.js.
https://github.com/OkoyeCharles/math-magicians/blob/1deb486ca2ca5bba0febbd326b6a3f7fa5e0abd5/src/App.js#L1-L20

Please cross-check what branch you are on, you should be on the [calculator branch](https://github.com/OkoyeCharles/math-magicians/tree/calculator)

I'm sorry for giving you trouble but I a positively sure that I got that correct.